### ISSUE TITLE
[Accessibility][HC]: The no row-data area’s background is black in DGV on HC 1/Black theme

### ISSUE THREAD

**Issue by vladimir-krestov** [dotnet/winforms#1976]
* .NET Core Version: 3.1.0-preview1.19458.7
* Have you experienced this same bug with .NET Framework?: Yes

**Problem description:**
- DataGridView no row-data area color is not contrast

**Actual behavior:** 
The DGV no row-data area’s background is black in HC 1/Black themes.
HC 1:
![image](https://user-images.githubusercontent.com/49272759/65661769-2f229a80-e03b-11e9-9668-e897a4093cda.png)

HC Black:
![image](https://user-images.githubusercontent.com/49272759/65661774-334eb800-e03b-11e9-8af7-46ba55e18960.png)

**Expected behavior:**
The no row-data area’s background should be visible like it works in HC 2/White themes. 
HC 2:
![image](https://user-images.githubusercontent.com/49272759/65661794-3ea1e380-e03b-11e9-8057-486e3e02c1ce.png)

HC White:
![image](https://user-images.githubusercontent.com/49272759/65661800-45c8f180-e03b-11e9-89af-a01aa77a2d43.png)

**Minimal repro:**
- Create .Net Core app
- Add DataGridView to the app
- Check DataGridView background colors in High Contrast themes

**Comment by Marie-Shi** [comment]
@vladimir-krestov Verified this bug with .Net Core 3.1.100-preview2-014533 from Release/3.1.1 branch, this bug is fixed. Please see the following screenshots:
HC Black:
![HC_Black](https://user-images.githubusercontent.com/45864017/67465652-22b05280-f678-11e9-9708-1a9d0e529e50.png)

HC White:
![HC_White](https://user-images.githubusercontent.com/45864017/67465786-6c00a200-f678-11e9-8d3e-9a466c9a895c.png)

HC #1: 
![HC_#1](https://user-images.githubusercontent.com/45864017/67465671-2c39ba80-f678-11e9-8c93-37e5c740cec8.png)

HC #2: 
![HC_#2](https://user-images.githubusercontent.com/45864017/67465678-2f34ab00-f678-11e9-9c42-bc5f85c0036f.png)

**Comment by vladimir-krestov** [comment]
Thank you!
